content/en/docs/demo/features.md

```diff
@@ -4,25 +4,24 @@ linkTitle: Features
 aliases: [/docs/demo/demo_features]
 ---
 
-- **[Kubernetes](https://kubernetes.io)**: the app is designed to run on
+- **[Kubernetes](https://kubernetes.io/)**: the app is designed to run on
   Kubernetes (both locally, as well as on the cloud) using a Helm chart.
 - **[Docker](https://docs.docker.com)**: this forked sample can also be executed
   only with Docker.
-- **[gRPC](https://grpc.io)**: microservices use a high volume of gRPC calls to
+- **[gRPC](https://grpc.io/)**: microservices use a high volume of gRPC calls to
   communicate to each other.
 - **[HTTP](https://www.rfc-editor.org/rfc/rfc9110.html)**: microservices use
   HTTP where gRPC is unavailable or not well supported.
-- **[OpenTelemetry Traces](https://opentelemetry.io)**: all services are
+- **[OpenTelemetry Traces](/docs/concepts/signals/traces/)**: all services are
   instrumented using OpenTelemetry available instrumentation libraries.
-- **[OpenTelemetry Metrics](https://opentelemetry.io)**: Select services are
-  instrumented using OpenTelemetry available instrumentation libraries. More
+- **[OpenTelemetry Metrics](/docs/concepts/signals/metrics/)**: Select services
+  are instrumented using OpenTelemetry available instrumentation libraries. More
   will be added as the relevant SDKs are released.
-- **[OpenTelemetry Collector](/docs/collector/getting-started)**: all services
-  are instrumented and sending the generated traces and metrics to the
-  OpenTelemetry Collector via gRPC. The received traces are then exported to the
-  logs and to Jaeger; received metrics and exemplars\* are exported to logs and
-  Prometheus.
-- **[Jaeger](https://www.jaegertracing.io)**: all generated traces are being
+- **[OpenTelemetry Collector](/docs/collector/)**: all services are instrumented
```

Comment:
Note the change in path from the Collector Getting-started page to the Collector landing page.